modules/protect/protect-dashboard-widget.css

```diff
@@ -114,3 +114,16 @@
 	margin-top: -30px;
 	padding: 0 12px;
 }
+
+.accessibility-visually-hidden { 
```

Comment:
WP core uses `a11y` instead of `accessibility` when prefixing variables. It's shorter.